libraries/entities/src/EntityScriptingInterface.h

```diff
@@ -110,9 +111,11 @@ class ParabolaToEntityIntersectionResult {
  * displayed and so knows about. For assignment client scripts, the entities available are those that are "seen" by the 
  * {@link EntityViewer}. For entity server scripts, all entities are available.</p>
  *
- * <h3>Entity Types</h3>
+ * <h3>Entity Types & Properties</h3>
  *
  * <p>For a list of the entity types that you can use, see {@link Entities.EntityType|Entity Types}.</p>
+ * 
+ * <p>While each entity type has properties that are specific to it, there are properties that are universal to all entity types, see {@link Entities.EntityProperties|Entity Properties}.</p>
```

Comment:
```suggestion
 * <p>For the properties of the different entity types, see {@link Entities.EntityProperties|Entity Properties}. Some properties are universal to all entity types, and some are specific to particular entity types.</p>
```

Reply:
I feel like this is... incorrect? My link should be to the universal entity properties, however the text implies that you are going to a list of entity-type-specific properties.

Reply:
The table of common properties has a list of links to the entity-specific properties, under the table. The description of the table says this ...
```
Different entity types have different properties: some common to all entities (listed in the table) and some specific to each EntityType (linked to below).
```
I.e., the common properties table is a good way in to discover all entity properties.

Reply:
Ah, alright.